auslib/web/admin/swagger/api.yaml

```diff
@@ -108,6 +131,110 @@ parameters:
     minimum: 0
     required: true
 
+  change_id_param:
+    name: change_id
+    in: path
+    description: Unique id to identify the identify the change
+    type: string
```

Comment:
Nope. [Here](https://github.com/mozilla/balrog/blob/master/auslib/web/admin/base.py#L116) it is defined as string type path parameter. For setting it as integer type we need to add int:change_id. Example for sc_id, [link](https://github.com/mozilla/balrog/blob/master/auslib/web/admin/base.py#L119)